src/components/PageSettingsModal/PageSettingsModal.jsx

```diff
@@ -122,13 +122,15 @@ export const PageSettingsModal = ({
                   To edit page content, simply click on the page title. <br />
                   <Breadcrumb params={params} title={watch("title")} />
                   {/* Title */}
-                  <FormField
-                    register={register}
-                    title="Page title"
-                    id="title"
-                    errorMessage={errors.title?.message}
-                    isRequired
-                  />
+                  <FormContext isRequired hasError={!!errors.title?.message}>
+                    <FormTitle>Page Title</FormTitle>
+                    <FormField
+                      placeholder="Page Title"
+                      id="title"
+                      {...register("title", { required: true })}
+                    />
```

Comment:
Nit: old one was `Page title` - but I'm fine with capitalisation also. The actual figma shows all titles as having capitalisation only on the first word (e.g. `Meta description` - we might want to modify the rest if we uncapitalise it. Alternatively, we should also capitalise `Page Details` if we swap to capitalisation

Reply:
i actually noticed this and thought that the capitalised version looks better but if it's in the figma then i'll change it! thanks for letting me know!